landing/tests.py

```diff
@@ -1,3 +0,0 @@
-# from django.test import TestCase
```

Comment:
This file is empty, if you don't use it please remove it

Reply:
> 

It is in the PR because I deleted it